lib/generators/graphiti/resource_generator.rb

```diff
@@ -94,7 +106,7 @@ def default_attributes
       end
       if attributes_class.table_exists?
         return attributes_class.columns.map do |c|
-          OpenStruct.new({name: c.name.to_sym, type: c.type})
+          OpenStruct.new({name: c.name.to_sym, type: c.type == :text ? :string : c.type})
```

Comment:
This line has enough colons in it where I think it's a little difficult to read. What do you think about deciding what the `type` will be on the previous line?

Reply:
I like it!

Reply:
What do you think now? I'm still using ternary, which is admittedly not the most readable, but I liked that more than: 
```ruby
column_type = c.type
column_type = :string if c.type == :text
```

What do you think?

Reply:
Hmm, I think either of those are fine. I'm kind of leaning towards the two line solution but both are readable enough for my tastes. I think if you're going to do a ternary operator (especially one with symbols) it needs to be on its own line.